downstream/modules/eda/con-decision-environments.adoc

```diff
@@ -0,0 +1,15 @@
+[id="decision-environments"]
+
+= Decision Environments
+
+
+{EDAName} includes, by default, an ansible.eda collection, which contains some sample sources, event filters and rulebooks. All the collections, ansible rulebooks and their dependencies use a Decision Environment, which is an image that can be run on either Podman or Kubernetes.
```

Comment:
Suggest removing the word "some", I don't think it's necessary. It's cleaner to state:
EDA includes, by default, an ansible.eda collection, which contains sample sources, event filters and rulebooks.

Reply:
Agree in this instance. Removing the word.